<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorHighlightingStylesPage</class>
 <widget class="QWidget" name="EditorHighlightingStylesPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>592</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="headerLabel">
     <property name="text">
      <string>&lt;b&gt;Configure syntax highlighting&lt;/b&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line1">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="TextLabel1_3">
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>Lexer Language:</string>
       </property>
       <property name="buddy">
        <cstring></cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="lexerLanguageComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Select the lexer language to be configured.</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="styleGroup">
     <property name="title">
      <string>Style Element</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" rowspan="12">
       <widget class="QListWidget" name="styleElementList">
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="foregroundButton">
        <property name="toolTip">
         <string>Select the foreground colour.</string>
        </property>
        <property name="text">
         <string>Foreground Colour</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="backgroundButton">
        <property name="toolTip">
         <string>Select the background colour.</string>
        </property>
        <property name="text">
         <string>Background Colour</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="fontButton">
        <property name="toolTip">
         <string>Select the font.</string>
        </property>
        <property name="text">
         <string>Font</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="eolfillCheckBox">
        <property name="toolTip">
         <string>Select end of line fill.</string>
        </property>
        <property name="text">
         <string>Fill to end of line</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="defaultButton">
        <property name="toolTip">
         <string>Press to set the current style to its default values</string>
        </property>
        <property name="text">
         <string>to Default</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <spacer>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>51</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="7" column="1">
       <widget class="QPushButton" name="allBackgroundColoursButton">
        <property name="toolTip">
         <string>Select the background colour for all styles</string>
        </property>
        <property name="text">
         <string>All Background Colours</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QPushButton" name="allFontsButton">
        <property name="toolTip">
         <string>Select the font for all styles.</string>
        </property>
        <property name="text">
         <string>All Fonts</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QPushButton" name="allEolFillButton">
        <property name="toolTip">
         <string>Select the eol fill for all styles</string>
        </property>
        <property name="text">
         <string>All Fill to end of line</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QPushButton" name="allDefaultButton">
        <property name="toolTip">
         <string>Press to set all styles to their default values</string>
        </property>
        <property name="text">
         <string>All to Default</string>
        </property>
       </widget>
      </item>
      <item row="12" column="0" colspan="2">
       <widget class="QLineEdit" name="sampleText">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>Sample Text</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignHCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="13" column="0" colspan="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>&lt;b&gt;Note:&lt;/b&gt; The tick in the list above indicates the entrie's 'fill to end of line' setting.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="importCurrentButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Imports all styles of the currently selected language</string>
       </property>
       <property name="text">
        <string>Import styles</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exportCurrentButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Exports all styles of the currently selected language</string>
       </property>
       <property name="text">
        <string>Export styles</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="importAllButton">
       <property name="toolTip">
        <string>Imports all styles of all languages</string>
       </property>
       <property name="text">
        <string>Import all styles</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exportAllButton">
       <property name="toolTip">
        <string>Exports all styles of all languages</string>
       </property>
       <property name="text">
        <string>Export all styles</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lexerLanguageComboBox</tabstop>
  <tabstop>styleElementList</tabstop>
  <tabstop>foregroundButton</tabstop>
  <tabstop>backgroundButton</tabstop>
  <tabstop>fontButton</tabstop>
  <tabstop>eolfillCheckBox</tabstop>
  <tabstop>defaultButton</tabstop>
  <tabstop>allBackgroundColoursButton</tabstop>
  <tabstop>allFontsButton</tabstop>
  <tabstop>allEolFillButton</tabstop>
  <tabstop>allDefaultButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
